bootstrap.js

```diff
@@ -4,42 +4,49 @@
 
 Components.utils.import("resource://gre/modules/Services.jsm");
 
-function startup(data, reason)
-{
-  Services.prefs.setBoolPref("dom.vr.enabled", true);
+const PREFS = {
+  // Enables WebVR. Not needed for Nightly, but we need to keep for other versions.
+  "dom.vr.enabled": true,
```

Comment:
I highly recommend against setting dom.vr.add-test-devices to 1, as this effectively disables the only way for web authors to detect that you have any VR hardware.